### ISSUE TITLE
Give VoiceOver clue when stats starts and finishes loading

### ISSUE THREAD

**Issue by astralbodies** [wordpress-mobile/WordPressCom-Stats-iOS#235]

**Comment by astralbodies** [comment]
Moving to the next release. I need to get more clarification from an actual blind user as to what the experience should be like.

**Comment by astralbodies** [comment]
Closing issue and migrating to new repo.
